meshing_plugins/include/meshing_plugins.h

```diff
@@ -0,0 +1,27 @@
+#ifndef MESHING_PLUGINS_H
+#define MESHING_PLUGINS_H
+
+#include <meshing_base.h>
+#include <pcl/surface/concave_hull.h>
+#include <pcl/surface/ear_clipping.h>
+#include <geometry_msgs/PoseArray.h>
+
+const static double CONCAVE_HULL_ALPHA = 0.1;
+
+namespace meshing_plugins
+{
+  class ConcaveHullMesher : public meshing_base::MeshingBase
+  {
+  private:
+    pcl::ConcaveHull<meshing_base::Point> chull_;
+    pcl::EarClipping clipping_;
```

Comment:
Have you considered just making the `chull_` objects and `clipping_` objects local variables inside the `generateMesh` function?